cmd/lint/main.go

```diff
@@ -160,7 +159,6 @@ var properNouns = []string{
 	"ID",
 	"Kafka",
 	"RBAC",
-	"REST",
```

Comment:
This is an english word that should sometimes be lowercase. We shouldn't force it to always be capitalized.